R/universalmotif_df.R

```diff
@@ -201,7 +216,14 @@ update_motifs <- function(motif_df, extrainfo = FALSE) {
       }
     }
   }
-  to_df(m, extrainfo)
+  if (extrainfo & all(!is.na(extrainfo_holdout_cols))){
+    # Add back any heldout info
+    new_df <- to_df(m, extrainfo)
+    # TODO: update id_cols strategy
+    return(merge(new_df, extrainfo_holdouts, by = id_cols, all.x = TRUE))
```

Comment:
Can you wrap the returning object with `structure(..., class = c("universalmotif_df", "data.frame"))` here?

Reply:
Whoops! Forgot to do that. Fixed. 

I also added a new unit test to catch similar errors in the future.